packages/@aws-cdk/aws-iam/lib/role.ts

```diff
@@ -184,13 +185,20 @@ export class Role extends Resource implements IRole {
 
     class Import extends Resource implements IRole {
       public readonly grantPrincipal: IPrincipal = this;
+      public readonly principalAccount = roleAccount;
       public readonly assumeRoleAction: string = 'sts:AssumeRole';
       public readonly policyFragment = new ArnPrincipal(roleArn).policyFragment;
       public readonly roleArn = roleArn;
       public readonly roleName = roleName;
       private readonly attachedPolicies = new AttachedPolicies();
       private defaultPolicy?: Policy;
 
+      constructor(_scope: Construct, _id: string) {
```

Comment:
No underscores here.

Reply:
Underscores are needed here, because there's already `scope` and `id` in this block.